src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalSum = days * 40;
+  const discountMin = 20;
+  const discountMax = 50;
+
+  if (days < 3) {
```

Comment:
For days: `3` and `7`, create separate values, cause now they are 'magic numbers'